### PR TITLE
Adds more data to regression test of the text near edge detector

### PR DESCRIPTION
**Pull request by vladimir-nazarenko**
And also adds some minor improvements

### CODE REVIEW

File: tests/regression/text_near_edge/test_regression_text_near_edge.py [hunk 1]
```diff
@@ -4,31 +4,64 @@
...
+                img = cv2.imread(filename)
+                screenshot = Screenshot(img, None, None, None)
+                pr_calculator.expected(True).found(dtor.execute(screenshot))
+
+        for path in negative_paths:
+            filenames = glob(path + "*.png")
+            for filename in filenames:
+                img = cv2.imread(filename)
+                screenshot = Screenshot(img, None, None, None)
+                pr_calculator.expected(False).found(dtor.execute(screenshot))
```
Comment (amosov-f) [inline tests/regression/text_near_edge/test_regression_text_near_edge.py]: Тут копипаста кода выше с изменением только True на False

File: tests/__init__.py [hunk 2]
```diff
@@ -1 +1,3 @@
+import os
 
+DATA_ROOT = os.path.join(os.path.dirname(__file__), "..", "data")
```
Comment (amosov-f) [inline tests/__init__.py]: Лучше сделать базовый класс для всех тестов и пихнуть туда такой метод (и ему подобные)

Comment (vladimir-nazarenko) [reply tests/__init__.py]: А чем именно это лучше?
